charts/proof-of-sats-plugin/templates/deployment.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "proof-of-sats-plugin.fullname" . }}
+  labels:
+    {{- include "proof-of-sats-plugin.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
```

```suggestion
  replicas: {{ .Values.replicaCount }}
```

I don't think we need the autoscaling block